### PR TITLE
Improvement/container image/latest

### PR DESCRIPTION
**Pull request by SgtSilvio**
https://hivemq.kanbanize.com/ctrl_board/41/cards/21246/details/

### CODE REVIEW

**Review by LukasBrand — APPROVED** [review]

File: hivemq4/base-image/entrypoints.d/30_configure_permissions.sh [hunk 1]
```diff
@@ -1,20 +0,0 @@
-#!/usr/bin/env bash
-
-# Step down from root privilege, only when we're attempting to run HiveMQ though.
-if [[ "$1" = "/opt/hivemq/bin/run.sh" && "$(id -u)" = '0' && "${HIVEMQ_NO_ROOT_STEP_DOWN}" != "true" ]]; then
```
Comment (ymengesha) [inline hivemq4/base-image/entrypoints.d/30_configure_permissions.sh:4]: There are some references about this root step down in the readme file. Let's also clean that up.

File: hivemq4/dns-image/Dockerfile [hunk 2]
```diff
@@ -3,6 +3,8 @@ ARG BASEIMAGE=hivemq/hivemq4:latest
 
 FROM ${BASEIMAGE}
 
+USER 0
```
Comment (ymengesha) [inline hivemq4/dns-image/Dockerfile:6]: curious, for what operation are we switching to the root user?

**Review by ymengesha — APPROVED** [review]
Non-blocking comments.

File: hivemq4/dns-image/Dockerfile (hunk 2 above)
Comment (SgtSilvio) [reply hivemq4/dns-image/Dockerfile:6]: for example `chmod` commands

File: hivemq4/base-image/entrypoints.d/30_configure_permissions.sh (hunk 1 above)
Comment (SgtSilvio) [reply hivemq4/base-image/entrypoints.d/30_configure_permissions.sh:4]: Removed the sections from the readme

**Pull request merged**